internal/testgen/main.go

```diff
@@ -135,3 +136,13 @@ func genGoFileFromTmpl(output string, exec Executor, data any) error {
 
 	return os.WriteFile(output, formatted, 0o644)
 }
+
+func sortAssertions(assertions []Assertion) {
```

Comment:
move it to `assertion.go`

let's keep `main.go` free from any helpers, the file is about generator itself

Reply:
Sure. Thanks for the review